gapis/resolve/framebuffer_attachment.go

```diff
@@ -39,7 +39,9 @@ func FramebufferAttachment(
 	attachment api.FramebufferAttachment,
 	settings *service.RenderSettings,
 	hints *service.UsageHints,
+	config *path.ResolveConfig,
```

Comment:
This a bit redundant now. Both the `ReplaySettings` and the `ResolveConfig` contain the replay device.

Should we also remove the default device selection below?

Reply:
I want to move this RPC into a Get. At that point, yes, we should certainly de-dupe this field.
I haven't done this yet as I wanted to make sure these changes worked first.

Reply:
SGTM